docs/TRIBE.md

```diff
@@ -60,6 +60,9 @@ Name 	    Number of Members 	 plugins 	 tasks
 all-nodes 	1 			         0   		 0
 ```
 
+#### Quick example for creating and joining agreements:
```

Comment:
@mkleina, what do you think about adding a section `Examples` including:
 - Starting a 4 node cluster and listing members
 - Creating an agreement and joining members to it 
 - Loading plugins  on a node participating in an agreement 
 - Listing metrics on all nodes participating in an agreement
    (Note: after loading a collector and publishing plugin on one node
 we demonstrate that these plugins are available on all of the other nodes in the agreement)
- Starting a task on a node participating in an agreement 
(Note: after starting a task on one node we demonstrate that tasks are now running on all of the other nodes in the agreement)